crates/nu-command/src/path/expand.rs

```diff
@@ -34,6 +35,11 @@ impl Command for SubCommand {
                 "Throw an error if the path could not be expanded",
                 Some('s'),
             )
+            .switch(
+                "no-deference",
```

Comment:
not sure what no-deference is. maybe you meant no-dereference?

Reply:
> maybe you meant no-dereference

Yeah, I just copy from `cp` command: https://github.com/nushell/nushell/blob/main/crates/nu-command/src/filesystem/cp.rs#L60

Reply:
I'd vote for `no-follow-link`

Reply:
I'm sorry but `no-follow-link` just sounds bad too. I think `dont-follow-link` sounds better, even if we leave the apostrophe out. What do you think?